blueprints/component-class/index.js

```diff
@@ -14,7 +14,6 @@ const OCTANE = has('octane');
 // TODO: this should be reading from the @ember/canary-features module
 // need to refactor broccoli/features.js to be able to work more similarly
 // to https://github.com/emberjs/data/pull/6231
```

Comment:
Assume this should be removed as well?

Reply:
Yeah, so as `this.EMBER_GLIMMER_SET_COMPONENT_TEMPLATE` below should be removed entirely as well